scripts/log_analytics.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function, absolute_import
```

Comment:
This file should not go into the final PR, but is here to document my approach for testing query times for deeply nested projects